src/PackageCompiler.jl

```diff
@@ -1,6 +1,7 @@
 module PackageCompiler
 
 using Base: active_project
+using REPL
```

Comment:
fyi it looks like REPL isn't being used in the merged version of this PR, so the dep could probably be dropped